src/main/java/seedu/address/logic/commands/add/AddMemberCommand.java

```diff
@@ -10,7 +10,7 @@
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Member;
+import seedu.address.model.person.member.Member;
 
```

Comment:
When running the app, for the AddMember command, the prompt does not indicate the existence of or how to input loyalty points.
This is what is being seen now:
addmember: Adds a member to the restaurant book. Parameters: n/NAME p/PHONE e/EMAIL
Example: addmember n/John Doe p/98765432 e/johnd@example.com